webdriverio/appium-app/examples/simple-example/test/configs/wdio.shared.conf.ts

```diff
@@ -0,0 +1,21 @@
+export const config: WebdriverIO.Config = {
```

Comment:
can you also provide a link to where people can find the main config and the options?